__tests__/app.test.js

```diff
@@ -216,4 +216,52 @@ describe("/api/articles/:article_id/comments", () => {
             expect(body.message).toBe("Missing Information")
         })
     })
+})
+
+describe("/api/articles/:article_id", () => {
+    test("PATCH: 200 - should respond with an updated article with changed number of votes", () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({inc_votes: 10})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.votes).toBe(110)
+        })
+    })
```

Comment:
You could put in a test to check the votes decrement accordingly - ie:` { inc_votes : -100 }`